cl/scrapers/management/commands/cl_report_scrape_status.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Why do we need this? Is it just a performance enhancement to put it everywhere?

Reply:
I don't know about "performance enhancement". It's not there for performance, but rather linting and type-hinting.

Overall, I like to import it whenever I'm type-hinting. [It's probably a best practice, especially for future changes.](https://docs.python.org/3/whatsnew/3.7.html#pep-563-postponed-evaluation-of-annotations)


I think this code will run without it. I can get rid of it if you'd like. @mlissner

Reply:
Yeah, the `__future__` gives me nightmare flashbacks from Py2. Let's remove it, since it looks like it'll be the default in version soon anyway.

Reply:
@mlissner Now it's done.